cli/tools/registry/pm.rs

```diff
@@ -267,9 +267,8 @@ pub async fn add(
   let is_npm = config_file.is_npm();
   for selected_package in selected_packages {
     log::info!(
-      "Add {} - {}@{}",
-      crate::colors::green(&selected_package.import_name),
-      selected_package.package_name,
+      "Add {}@{}",
+      crate::colors::green(&selected_package.package_name),
```

Comment:
can it be the resolved package version?

Reply:
PTAL:
<img width="644" alt="Screenshot 2024-08-08 at 15 28 51" src="https://github.com/user-attachments/assets/29ec3889-311f-4d2b-b52f-4b480ef04608">

Reply:
Great!